pages/search/index.vue

```diff
@@ -61,6 +95,39 @@
 
     mounted() {
       this.$store.commit('search/enableCollectionFacet');
+      if (this.$route.query.query.length > 0) {
+        this.getSearchSuggestions(this.$route.query.query);
+      }
+    },
+
+    methods: {
+      async getSearchSuggestions(query) {
+
+        // Query in the user's language, removing duplicates
+        const languageParam = Array.from(new Set([this.$i18n.locale])).join(',');
+
+        const suggestions = await getEntitySuggestions(query, {
+          language: languageParam
+        }, {
+          recordValidation: this.enableSuggestionValidation
+        });
+
+        this.relatedCollections = suggestions.slice(0, 4);
+      },
+
+      suggestionLinkGen(entityUri) {
+        const entity = {
+          id: entityUri,
+          prefLabel: this.relatedCollections[entityUri]
```

Comment:
As discussed, this seems to be always empty. Could you see if you can change that?

Reply:
now fixed